### DISCUSSION TITLE
Feature : add local attachments to note

### DISCUSSION THREAD

**Discussion by mfaessel-CCPS** [1943time/inkdown#88] [Ideas]
Hi,
Would be nice to add attachments to a note PDF ZIP, DOC, ...
Regards !

**Comment by 1943time** [comment]
The next version can add attachments, but cannot directly display PDF or other formats. As shown in the following image:

<img width="719" alt="image" src="https://github.com/1943time/inkdown/assets/115093678/dea65a99-7a0e-427e-8d75-32b44c5ba342">

**Comment by mfaessel-CCPS** [comment]
Happy to see this on the next version !
Thank you !
